tests/suites/sidecar/task.sh

```diff
@@ -12,6 +12,7 @@ test_sidecar() {
 		test_deploy_and_force_remove_application
 		test_pebble_notices
 		test_pebble_checks
+		test_credential_get_k8s
```

Comment:
Are these sidecar tests automatically run in CI, and will this one be picked up automatically?

Reply:
Merged now. Please regenerate!